fishtest/fishtest/views.py

```diff
@@ -347,7 +352,7 @@ def tests_run(request):
       run_id = request.rundb.new_run(**data)
 
       request.actiondb.new_run(authenticated_userid(request), request.rundb.get_run(run_id))
-      last_time = 0
+      clear_cache()
       request.session.flash('Started test run!')
```

Comment:
Can't you just manually decorate `request.session.flash`?

I mean to create a wrapper function:

```
def cached_flash(request, requestString):
  clear_cache()
  request.session.flash(requestString)
  return 
```
I think it would make the code clearer.

Reply:
Excellent idea!